src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -662,3 +663,16 @@ def get_release_images_path() -> str:
         return f"{consts.ASSISTED_SERVICE_DATA_BASE_PATH}/default_{flavor}_release_images.json"
 
     return f"{consts.ASSISTED_SERVICE_DATA_BASE_PATH}/default_release_images.json"
+
+
+def return_yaml_from_string(string: str) -> str:
+    """format online string to yaml
+    input example: 'dns-resolver:\n  config:\n    server:\n    - 192.168.127.1`
+    output example: '
+        dns-resolver:
+          config:
+            server:
+            - 192.168.127.1
+    '
+    """
+    return re.sub(r"\\.", lambda x: {"\\n": "\n", "\\t": "\t"}.get(x[0], x[0]), string)
```

Comment:
looks like the python lib has the tooling to to it (from https://codewithanbu.com/how-to-un-escape-a-backslash-escaped-string-with-python/):
```
>>> s = "dns-resolver:\\n  config:\\n    server:\\n    - 192.168.127.1"
>>> print(s)
dns-resolver:\n  config:\n    server:\n    - 192.168.127.1
>>> print(s.encode('raw_unicode_escape').decode('unicode_escape'))
dns-resolver:
  config:
    server:
    - 192.168.127.1
```

Reply:
I tried it, it didn'r work well here and I had issue sterilizing  the yaml

I'll take a look on you suggestion ( I rather not use `re` )